metricbeat/docs/modules/system.asciidoc

```diff
@@ -265,6 +265,11 @@ metricbeat.modules:
 
   # Filter systemd services based on a name pattern
   #service.pattern_filter: ["ssh*", "nfs*"]
+
+  # This option enables the use of performance counters to collect data for cpu/core metricset.
+  # Only effective for Windows.
+  # You should use this option if running beats on machins with more than 64 cores.
+  #use_performance_counters: false
```

Comment:
We expect to change to default to true once we are confident there are no regressions, correct?

Is it worth just defaulting it to true now, and only turning it off if there is a problem?

This won't be released until 9.0 and 8.18.0 given the current timing.